rust/headless-client/src/lib.rs

```diff
@@ -100,7 +100,6 @@ pub enum IpcServerMsg {
         error_msg: String,
         is_authentication_error: bool,
     },
-    OnTunnelReady,
```

Comment:
`OnTunnelReady` was redundant, the first `OnUpdateResources` happens after `OnTunnelReady` and can be used as the "tunnel is ready" signal.